admin_panel/users/viewsets/employee.py

```
@@ -0,0 +1,119 @@
+from rest_framework import viewsets, filters
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from admin_panel.users.models.employee import Employee, EmployeeRole
+from admin_panel.users.serializers.employee import EmployeeResponseSerializer, EmployeeSerializer
+
+
+from rest_framework import viewsets, filters
```

Ruff (0.7.0)

Redefinition of unused `viewsets` from line 1

Remove definition: `viewsets`

(F811)